### PR TITLE
Add projects and rename EF category to IE

### PR DESCRIPTION
**Pull request by joshhopkins**
**Insert URLs to the project(s) here:** 
- https://git.tu-berlin.de/genesysmod/genesys-mod-public
- https://github.com/fineprint-global
- https://github.com/konstantinstadler/pymrio
- https://github.com/CMLPlatform/ramascene
- https://github.com/CMLPlatform/CircuMAT
- https://github.com/byrd-polar/fluid-earth
- https://github.com/TNO/EXIOMOD-open

**Explain what this list is about and why it should be included here:**     
Active projects widely used by the community.

**The community behind OpenSustain.tech offers free marketing and consulting for open source projects added to the list. If you are interested just tick here:**

- [x]  Share the project on all available social media channels. ( Reddit, Twitter, Linkedin, HackerNews ). 
- [ ]  Collaborative blog post about the project with publication on OpenSustain.tech and OpenSource.com. 
- [ ]  Get feedback on your project. 
- [ ]  Guidance on further development strategies and funding for the project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1303,6 +1310,7 @@ If you're looking for an entry point into domains like renewable energy, climate
 - [Terraso](https://github.com/techmatters/terraso-backend) - Develop an open source technology platform to help local community leaders practice Integrated Landscape Management.
 - [Open Soil Spectral Library](https://soilspectroscopy.github.io/ossl-manual/) - Brings together soil scientists, spectroscopists, informaticians, data scientists and software engineers to overcome some of the current bottlenecks preventing wider and more efficient use of soil spectroscopy.
 
+- [GainForest](https://github.com/GainForest) - A decentralised fund using artificial intelligence to measure and reward sustainable nature stewardship.
```
Comment (joshhopkins) [inline README.md:1312]: Might not want to include at this time.

Comment (Ly0n) [reply README.md:1312]: Yes, as long as no open source project is released that fits to the guideline I would not include it.

**Review by Ly0n — APPROVED** [review]
The EXIOMOD project is pretty much on the edge in terms of recent activity. The lack of issues suggests that the actual development repo is internal and they only publish it here.

**Comment by Ly0n** [comment]
Thank you for your pull request. I'm happy to welcome you as an official contributor to the project in addition to the hints and tips you've given me in direct conversations so far. 

The projects you added are a really good fit.

File: README.md [hunk 2]
```diff
@@ -713,7 +714,8 @@ If you're looking for an entry point into domains like renewable energy, climate
 - [premise](https://github.com/romainsacchi/premise) - Coupling Integrated Assessment Models output with Life Cycle Assessment.
 - [useeior](https://github.com/USEPA/useeior) - Estimating potential environmental impacts of goods and services in the US economy.
 - [fedelemflowlist](https://github.com/USEPA/Federal-LCA-Commons-Elementary-Flow-List) - A Python package that generates and provides a standardized elementary flow list for use in life cycle assessment (LCA) data as well as mappings to convert data from other sources.
-
+- [FORBIO](https://github.com/fineprint-global) - Physical multi-regional input-output model for forestry, wood and paper products.
```
Comment (Ly0n) [inline README.md:717]: The link must point directly to the repo of the projects. If an organization has multiple project worth listing that we list them all. The data mining script that is creating the database needs this.

Comment (joshhopkins) [reply README.md:717]: Whoops, meant to be https://github.com/fineprint-global/forbio

**Pull request merged**